### ISSUE TITLE
Login page fails HTML Validation

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#26]
URL: https://hcdev.openplus.ca/en/user/login
User Agent: Firefox
Language: English

**Issue Description:** Page fails HTML validation.
**Steps to reproduce:**
1. Generate source code
2. Copy code
3. in a 2nd tab, open [W3C HTML Validator - Validate by Direct input tab](https://validator.w3.org/#validate_by_input)
4. paste code in text area and click "Check"
list of errors will be displayed. 
![image](https://user-images.githubusercontent.com/65728700/128040888-c91c1232-60f7-48e4-900c-19ad1488cc40.png)

Note the Reset Password page has errors as well. 

**Expected results:**
Fix validation errors. 
**All pages must pass HTML validation.**

**Comment by davidgotlieb** [comment]
The only error reported is `autocorrect=none' on the form field.
﻿If we remove this it could have negative consequences on other fields. The reason it is on the login page is so that autocorrect does not try to fix names. IE Trying to turn Craig into Crag.
﻿
﻿The rest are all warnings for minor things.

**Comment by hydegeoff** [comment]
Other pages needed to be verified as well, for example the dashboard fails HTML validator:

There are a few errors that we would like you to investigate:
![Dashboard-W3C HTML Validation failed](https://user-images.githubusercontent.com/88147013/159990562-4e50343a-433e-4be0-bb20-00f278b1799c.jpg)

**Comment by echouin** [comment]
Where are we on this wrt the Prod version? Thx!
